fawltydeps/cli_parser.py

```diff
@@ -163,8 +163,9 @@ def populate_parser_paths_options(parser: argparse._ActionsContainer) -> None:
         metavar="PYENV_DIR",
         help=(
             "Where to find Python environments that have project dependencies"
-            " installed. When empty (the default), fall back to the Python"
-            " environment where FawltyDeps is installed."
+            " installed. Defaults to looking for Python environments under the"
+            " current directory, or failing that, using the Python environment"
```

Comment:
Basepath. To be technically pedantic about it:

"Where to find Python environments that have project dependencies installed. Defaults to looking for Python environments under the given basepath(s), which again defaults to the current directory. If no Python environments are found under basepath(s), the Python environment where FawltyDeps is installed will be used."

We need some refactoring of the help messages here, because this is too long/complicated IMHO, and the `--code` and `--deps` options have the same problem...

Reply:
Maybe like that:
> Path to search for Python environments that have project dependencies installed. Default search order:\n
> - basepath(s) (by default the current directory)
> - Python environment where FawltyDeps is installed.